### PR TITLE
Bugfix/markdown

### PR DESCRIPTION
**Pull request by bbaassssiiee**
## Markdown linted
    
*mdl* can be used for a consistent style and proper rendering of Markdown files.
    
* To use it you need to install a Ruby gem and run mdl on a markdown file like README.md
    
    ```
    sudo gem install mdl
    mdl README.md
    ```

### CODE REVIEW

File: facilitator/solutions/tower_install.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-7. Sign in as the admin user with the password created in the installation process. You will be prompted for a license.
-8. Request a trial license if one hasn't been provided already. (Either trial license type will suffice.)
-9. Import/upload the license file.
-10. Extra Credit: Open the ping API end point in a web browser -- the URL will be something like like https://tower.host.ip/v1/api/ping.
+1. Run the setup.sh script with sudo.
+1. Verify Ansible has been successfully installed and started, load the Ansible Tower UI in a web browser using the control machine IP address.
+1. Sign in as the admin user with the password created in the installation process. You will be prompted for a license.
+1. Request a trial license if one hasn't been provided already. (Either trial license type will suffice.)
+1. Import/upload the license file.
+1. Extra Credit: Open the ping API end point in a web browser -- the URL will be something like like <https://tower.host.ip/v1/api/ping/>.
```
Comment (rubbsdecvik) [inline facilitator/solutions/tower_install.md]: Is there a reason not to used the `ordered` option with this? [docs](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md029---ordered-list-item-prefix) One advantage to the ordered option is that it is readable in plain text.

Comment (bbaassssiiee) [reply facilitator/solutions/tower_install.md]: An ordered list is prone to be reordered after a while. This style allows focus on the content and forgetting about keeping order manually.

Comment (rubbsdecvik) [reply facilitator/solutions/tower_install.md]: Good point. I am convinced.

File: facilitator/solutions/roles.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 # Workshop: Roles
 
-Here students are tasked with refactoring their previous work from the Basic Playbook workshop into a role and modifying their playbook accordingly. We intentionally avoid introducing any new tasks or functionality otherwise. The objective here is to focus students specifically on how roles are structured and develop. 
+Here students are tasked with refactoring their previous work from the Basic Playbook workshop into a role and modifying their playbook accordingly. We intentioinally avoid introducing any new tasks or functionality otherwise. The objective here is to focus students specifically on how roles are structured and develop.
```
Comment (gundalow) [inline facilitator/solutions/roles.md]: Typo intentioinally

File: facilitator/solutions/simple_playbook.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # Workshop: Simple Playbook
 
-This assignment provides a quick introduction to playbook structure to give them a feel for how Ansible works, but in practice is too simplistic to be useful. 
+This assignment provides a quick introduction to playbook structure to give them a feel for how Ansible works, but in pratice is too simplistic to be useful.
```
Comment (gundalow) [inline facilitator/solutions/simple_playbook.md]: typo pratice

File: facilitator/solutions/tower_install.md [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
 
 ### Solution
 
-1. Retrieve the latest version of Tower from the Ansible Tower releases archive.  
-2. Unarchive the retrieved Ansible Tower archive.
-    * i.e. `tar -vxzf towerlatest` 
-3. Change directories to the one created by the expanded Tower software archive.
-4. Edit the `inventory` file with passwords for admin, message queue and the postgres database will use while setting up. These are represented by these inventory variables:
-    * `admin_password` 
+1. Retreive the latest version of Tower from the Ansible Tower releases archive.
```
Comment (gundalow) [inline facilitator/solutions/tower_install.md]: typo Retrieve

File: facilitator/solutions/tower_install.md [hunk 5]
```diff
@@ -1,29 +1,30 @@
...
+1. Unarchive the retreived Ansible Tower archive.
```
Comment (gundalow) [inline facilitator/solutions/tower_install.md]: typo retrieved

File: facilitator/solutions/roles.md (hunk 2 above)
Comment (gundalow) [inline facilitator/solutions/roles.md]: typo intentionally

File: facilitator/solutions/simple_playbook.md (hunk 3 above)
Comment (gundalow) [inline facilitator/solutions/simple_playbook.md]: typo practice

File: facilitator/solutions/tower_install.md (hunk 5 above)
Comment (tima) [reply facilitator/solutions/tower_install.md]: We received a PR that was merged that fixes a lot of the mistakes which is probably why this PR that @bbaassssiiee submitted has conflicts and needs to be resolved.

**Comment by bbaassssiiee** [comment]
I'll make a new PR

**Pull request closed without merging**